serverless.yml

```diff
@@ -236,3 +233,31 @@ functions:
     package:
       include:
         - functions/AutoRemediateEC-005.js
+    role: revokeSecurityGroupIngressRole
+
+resources:
+
+  Resources:
+
+    revokeSecurityGroupIngressRole:
+      Type: AWS::IAM::Role
+      Properties:
+        RoleName: RevokeSecurityGroupIngressRole
+        AssumeRolePolicyDocument:
+          Version: '2018'
+          Statement:
+            - Effect: Allow
+              Principal:
+                Service:
+                  - lambda.amazonaws.com
+              Action:
+                - sts:AssumeRole
+      Policies:
+        - PolicyName: RevokeSecurityGroupIngressPolicy
```

Comment:
To keep it consistent, can you please change RevokeSecurityGroupIngressPolicy to AutoRemediateEC-002Policy

Reply:
same with policy